### DISCUSSION TITLE
Examples for gRPC retries

### DISCUSSION THREAD

**Discussion by schwichti** [linkerd/linkerd2#6892] [Q&A]
Hi, 
the release notes of `edge-21.6.2` https://github.com/linkerd/linkerd2/releases/tag/edge-21.6.2 suggests that there are gRPC retries. Where can I find exemplary configurations for gRPC retries?

**Comment by olix0r** [comment]
@schwichti You can generate a `ServiceProfile` resource from your protobuf definitions as explained at https://linkerd.io/2.11/tasks/setting-up-service-profiles/ and then configure retries for your routes as described at https://linkerd.io/2.10/tasks/configuring-retries/.

... and we should probably update the emojivoto examples to use include retries, too.
